### PR TITLE
Adding apps.json file

### PR DESCRIPTION
**Pull request by andretms**
Creating **apps.json** file  for Quickstarts

### CODE REVIEW

**Comment by andretms** [comment]
@danieldobalian, @kreedula - I have created the apps.json file to define the dependencies of the Quickstart. Can you please review it/ merge if you have a chance?

CC: @jmprieur

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an Android app",
+    "Level": 400,
```
Comment (jmprieur) [inline AppCreationScripts/apps.json]: Is it really a 400? or a 200?

File: AppCreationScripts/apps.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "x-ms-id": "active-directory-android-native-v2",
+      "x-ms-name": "android-native-v2",
+      "x-ms-version": "2.0",
+      "replyUrlsWithType": [
+        {
+          "url": "msal{AppId}://auth",
+          "type": "InstalledClient"
+        }
+      ],
+      "oauth2AllowImplicitFlow": true,
```
Comment (danieldobalian) [inline AppCreationScripts/apps.json]: Shouldn't set this to true for Auth code flow (mobile plats).

File: AppCreationScripts/apps.json (hunk 1 above)
Comment (danieldobalian) [reply AppCreationScripts/apps.json]: Definetly not 400. Probably 100, maybe 200.

**Pull request merged**